airflow/dags/kernel_changes.py

```diff
@@ -363,20 +387,18 @@ def register_orphan_issues(ds, **kwargs):
     """
 
     j_issues = kwargs["ti"].xcom_pull(key="j_issues", task_ids="register_journals_task")
+    orphan_issues = []
 
-    orphan_issues = Variable.get("orphan_issues", [])
-
-    for issue_id in orphan_issues:
-
-        resp_json = fetch_data(issue_id)
-
+    for issue_id in json.loads(Variable.get("orphan_issues", "[]")):
+        resp_json = fetch_bundles(issue_id)
         journal_id = [j for j, i in j_issues.items() if issue_id in i]
```

Comment:
Achei estranha esta linha, acho que deve haver jeito melhor para descobrir o journa_id. Se eu entendi bem... Talvez fora do for (linha 392), gerar um dicionário inverso, onde as chaves são issue_id e o valor journal_id.

Reply:
@robertatakenaka , essa dag precisa de uma refatoração completa, se vc olhar o código a fundo esta bem confuso mesmo 😕, so acho que vamos ter que fazer isso aos pouco